packages/nimble-components/CONTRIBUTING.md

```diff
@@ -18,13 +18,11 @@ The library is built on the open source [FAST Design System library](https://fas
 
 ## Getting started
 
-From the `nimble` directory:
-
-1. Run `npm install`
-2. Run `npm run build`
+1. Run `npm install` from the repo root directory
+2. Run `npm run build` from the `nimble` directory
```

Comment:
Do we have a `nimble` directory that's distinct from the repo root?  Any reason not to continue recommending `npm run build` from the repo root?

Reply:
+1 I'd advocate to have README tweaks in a follow-on PR to not have to cycle language revisions here

Reply:
Thanks, that should have said `nimble-components`. I thought it would be better if this "Getting started" was scoped to this project, but in retrospect, this project needs its dependencies built, so I'll change it.